install.sh

```diff
@@ -1,11 +1,31 @@
 #!/bin/bash
 
-SSID="${1:-WiFiAPi}"
-PASSPHRASE="${2:-Raspberrypi}"
-IP_RANGE="${3:-192.168.1}"
+SSID="${1:-RaspberryPi}"
+PASSPHRASE="${2:-raspberry}"
+IP_RANGE="${3:-192.168.222}"
 INC="${4:-eth0}"
 WIFI="${5:-wlan0}"
 
+pause() {
+#	read -p "Appuyer sur une touche pour continuer ..." var
+	echo -n
+}
+
+begin() {
+	echo -n $1
+}
+
+end() {
```

Comment:
Why are you adding this pause command everywhere? Please remove it and inline the `echo "done"` again.

Reply:
As I said just before, if you want to avoid displaying " done." or want to display anything else, you just have to change this function (and not to go to each inline echo to modify it).